pkgs/misc/drivers/hplip/default.nix

```diff
@@ -47,32 +47,27 @@ in
 assert withPlugin -> builtins.elem hplipArch pluginArches
   || throw "HPLIP plugin not supported on ${stdenv.system}";
 
-stdenv.mkDerivation {
+pythonPackages.mkPythonDerivation {
   inherit name src;
 
   buildInputs = [
     libjpeg
     cups
     libusb1
-    pythonPackages.python
-    pythonPackages.wrapPython
     sane-backends
     dbus
     net_snmp
     openssl
-  ] ++ stdenv.lib.optionals qtSupport [
-    qt4
   ];
 
   nativeBuildInputs = [
     pkgconfig
   ];
 
-  pythonPath = with pythonPackages; [
```

Comment:
Is their still any use for`pythonPath` or is it safe to assume that any use of this is deprecated?